connect_four/ai.rb

```diff
@@ -0,0 +1,38 @@
+require './board.rb'
+
+class AI
+  attr_reader :player_id
+  attr_accessor :game_board
+
+  def initialize(player_id, game_board)
+    @player_id = player_id
+    @game_board = game_board
+  end
+
+  def pick_column
+    rational_choice = pick_rationally
+    if rational_choice
+      rational_choice
```

Comment:
I wonder if there's a simpler way to write this other than if rational_choice return rational choice. Maybe not with the else.